website/index.html

```diff
@@ -91,9 +91,9 @@ <h1>Topiary</h1>
                 >
                     <div class="col-10 col-sm-8 col-lg-6">
                         <img
-                            src="images/jr-harris-MROejEmXGGE-unsplash.jpg"
+                            src="images/topiary_logo.png"
                             class="d-block mx-lg-auto img-fluid"
-                            alt="Cover photo by JR Harris on Unsplash"
+                            alt="Topiary logo by TODO"
```

Comment:
@ErinvanderVeen Would Maatje like credit here; she definitely deserves it :+1: If so, what would be best written here, to replace the current `TODO`?

Reply:
She would not like to be credited on the logo.